### PR TITLE
Add some key descriptions

### PR DESCRIPTION
**Pull request by glkappe**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.2 (TiDB 6.2 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [x] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by glkappe** [comment]
Since it is the minimum configuration, remove the description of safe-mode

File: migrate-aurora-to-tidb.md [hunk 1]
```diff
@@ -227,22 +227,21 @@ block-allow-list:                     # 如果 DM 版本早于 v2.0.0-beta.2 则
...
   - source-id: "mysql-01"               # 数据源 ID，即 source1.yaml 中的 source-id
     block-allow-list: "listA"           # 引入上面黑白名单配置。
-#       syncer-config-name: "global"    # 引用上面的 syncers 增量数据配置。
+    mydumper-config-name: "global"      # mydumpers 配置的名称
     meta:                               # task-mode 为 incremental 且下游数据库的 checkpoint 不存在时 binlog 迁移开始的位置; 如果 checkpoint 存在，则以 checkpoint 为准。
       binlog-name: "mysql-bin.000004"   # “Step 1. 导出 Aurora 快照文件到 Amazon S3” 中记录的日志位置，当上游存在主从切换时，必须使用 gtid。
       binlog-pos: 109227
       # binlog-gtid: "09bec856-ba95-11ea-850a-58f2b4af5188:1-9"
 
-   # 【可选配置】 如果增量数据迁移需要重复迁移已经在全量数据迁移中完成迁移的数据，则需要开启 safe mode 避免增量数据迁移报错。
```
Comment (lance6716) [inline migrate-aurora-to-tidb.md:240]: 上面的改动没问题，这个还是留还是不留，由 PM 判断一下吧。个人觉得这里确实不是必要的，但是给用户看一眼也没啥不好

File: migrate-aurora-to-tidb.md [hunk 2]
```diff
@@ -227,22 +227,21 @@ block-allow-list:                     # 如果 DM 版本早于 v2.0.0-beta.2 则
     - db-name: "test_db"              # 需要迁移的表的库名。
       tbl-name: "test_table"          # 需要迁移的表的名称。
 
+
+mydumpers:
```
Comment (lance6716) [inline migrate-aurora-to-tidb.md:231]: 这个是增量任务，没有必要设置全量阶段的配置

File: migrate-aurora-to-tidb.md [hunk 3]
```diff
@@ -227,21 +227,22 @@ block-allow-list:                     # 如果 DM 版本早于 v2.0.0-beta.2 则
...
-mydumpers: 
-  global:
-    extra-args: "--consistency none"    # # 对于 rds 没有 reload 权限，需要设置该参数避免锁表。
-
 # 配置数据源
 mysql-instances:
   - source-id: "mysql-01"               # 数据源 ID，即 source1.yaml 中的 source-id
     block-allow-list: "listA"           # 引入上面黑白名单配置。
-    mydumper-config-name: "global"      # mydumpers 配置的名称
+    syncer-config-name: "global"        # syncer 配置的名称
```
Comment (lance6716) [inline migrate-aurora-to-tidb.md:234]: 下面的被注释了，这里应该也要注释

```suggestion
#    syncer-config-name: "global"        # syncer 配置的名称
```

**Review by lance6716 — APPROVED** [review]

**Review by ran-huang — APPROVED** [review]

File: migrate-aurora-to-tidb.md [hunk 4]
```diff
@@ -231,18 +231,18 @@ block-allow-list:                     # 如果 DM 版本早于 v2.0.0-beta.2 则
...
       # binlog-gtid: "09bec856-ba95-11ea-850a-58f2b4af5188:1-9"
 
-   # 【可选配置】 如果增量数据迁移需要重复迁移已经在全量数据迁移中完成迁移的数据，则需要开启 safe mode 避免增量数据迁移报错。
+# 【可选配置】 如果增量数据迁移需要重复迁移已经在全量数据迁移中完成迁移的数据，则需要开启 safe mode 避免增量数据迁移报错。
    ##  该场景多见于以下情况：全量迁移的数据不属于数据源的一个一致性快照，随后从一个早于全量迁移数据之前的位置开始同步增量数据。
    # syncers:            # sync 处理单元的运行配置参数。
    #  global:           # 配置名称。
    #    safe-mode: true # 设置为 true，会将来自数据源的 INSERT 改写为 REPLACE，将 UPDATE 改写为 DELETE 与 REPLACE，从而保证在表结构中存在主键或唯一索引的条件下迁移数据时可以重复导入 DML。在启动或恢复增量复制任务的前 1 分钟内 TiDB DM 会自动启动 safe mode。
-
+```
```
Comment (ran-huang) [inline migrate-aurora-to-tidb.md:245]:
```suggestion
```

**Pull request merged**